serverlessworkflow/antora.yml

```diff
@@ -36,6 +36,7 @@ asciidoc:
     docker_min_version: 20.10.7
     docker_compose_min_version: 1.27.2
     # only used in downstream
+    operator_version: main
```

Comment:
what are we expecting here as a version ? the operator tag ?
we should update the [setup-branch file](https://github.com/kiegroup/kogito-docs/blob/main/.ci/jenkins/Jenkinsfile.setup-branch#L64) to update that one as well

Reply:
I updated the setup-branch and post-release files to set uo the right value for the operator_version variable.